src/components/plantDetails/PlantNav.tsx

```diff
@@ -1,41 +1,41 @@
 import PlantNavItem from "./PlantNavItem.tsx";
-import { Color } from "../colors.ts";
+import { theme } from "../../theme/theme.ts";
 import { PlantNavStyled } from "./PlantDetail.styles.ts";
 
 const PlantNav = () => {
   return (
     <PlantNavStyled>
       <PlantNavItem
-        backgroundColor={Color.lightGreen}
-        color={Color.mediumGreen}
+        backgroundColor={theme.colors.lightGreen}
+        color={theme.colors.mediumGreen}
         linkTo={"/monstera/podlewanie"} //the path will have to be dynamic
```

Comment:
is this list static or will it be dynamic? We talked about it before but the color repetition still strikes me as unnecessary. If PlantNavItem has to be generic (which it doesn't seem to judging by name) and accept many colors I'd create a new styled component just for this file with those colors defined by default.

Reply:
Good question. I think it will be dynamic... At least the path has to be. Fixed the problem with repetition and added background color and color to `PlantNavItemStyled`